### ISSUE TITLE
Warning: Each child in a list should have a unique "key" prop.

### ISSUE THREAD

**Issue by schwenzner** [gatsbyjs/gatsby#26962]
No code details are required. I was simply following the tutorial in the docs. I was at part 6, "Transformer Plugins". The code is copied as is in the docs. This is on Windows 10, using node 14.4, latest chrome and VS Code.
The warning is generated when attempting to use node.id  as key (please refer to last code displayed on the tutorial page for the file index.js).
Observations:
1. When changing the key to, for example, node.frontmatter.title it works and no warning appears.
2. The "console.log(data)" output in the browser shows the correct object data but without the id property of the node element even though it is listed clearly in the graphql query
3. The same query when played in QraphiQL clearly lists the id of each node !!!!
4. Attempting to use node.id in any statement results with undefined (obviously).

Again, what is puzzling here is the fact that the same query in GraphiQL displays the node id but when used in index.js it comes out missing..!
I have compiled and recompiled again and again but this seems to be persistent.

**Comment by arthurjdam** [comment]
Followed the tutorial and it works just fine: [https://github.com/arthurjdam/gatsby-issues-26962](https://github.com/arthurjdam/gatsby-issues-26962)
A code sample of your current state would help what's going on. A node's id is generated but needs to be requested. Is "node.id" a part of your GraphQL query?

**Comment by schwenzner** [comment]
Today is a new day...! and I was afraid something like this was going to happen. I turned on my PC to continue the tutorial and guess what?!, the console log output on the browser now had the id listed under each node

**Comment by schwenzner** [comment]
Today is a new day...! and I was afraid something like this was going to happen. I turned on my PC to continue the tutorial and guess what?!, the console log output on the browser now have the id listed under each node. I changed the key to reference the node.id and all is back to normal. I suspected somethin like this was going to happen. I regret, and apologize, for quickly jumping into adding this as an issue. I guess taking a break away from the machine helps both the coder and the machine...!!!!
